swing/src/net/sf/openrocket/gui/rocketfigure/RailButtonShapes.java

```diff
@@ -1,19 +1,28 @@
 package net.sf.openrocket.gui.rocketfigure;
 
 import java.awt.Shape;
-import java.awt.geom.Ellipse2D;
-import java.awt.geom.Line2D;
-import java.awt.geom.Path2D;
-import java.awt.geom.Point2D;
+import java.awt.geom.*;
```

Comment:
nit: its a matter of style (and your editor defaults like) - I personally think we should not use wildcard endpoints (nor does [google coding style guide](https://google.github.io/styleguide/javaguide.html#s3.3-import-statements) for that matter. It's a nit however as we don't have a standard defined